build_tools/python/benchmark_suites/iree/module_execution_configs.py

```diff
@@ -39,6 +39,12 @@ def _with_caching_allocator(
     loader=iree_definitions.RuntimeLoader.NONE,
     driver=iree_definitions.RuntimeDriver.CUDA)
 
+CUDA_CONFIG_LONG = _with_caching_allocator(
+    id=unique_ids.IREE_MODULE_EXECUTION_CONFIG_CUDA,
+    tags=["full-inference", "default-flags", "long-running"],
```

Comment:
We use the assumption that if "config_a.id = config_b.id", then "config_a" is identical to "config_b". So having two configs with the same id but different tags might run into some issues.